configfs/fakertmr/fakertmr.go

```diff
@@ -112,7 +112,7 @@ func writeTdx(entry string, attr string, content []byte, indexMap map[int]bool)
 			2: "8-15\n",
 			3: "\n",
 		}
-		tempTsmPathTcgMap := filepath.Join(os.TempDir(), tsmRtmrSubsystem, tsmPathTcgMap)
+		tempTsmPathTcgMap := filepath.Join(os.TempDir(), tsmPathTcgMap)
```

Comment:
This ought to be the provided root directory, no?

Reply:
`tcgmap` is read-only. Here we create a read-only file and move it to `t.TempDir()/rtmr/entry`. 

We can also create a read-only file under t.TempDir() but if so, we need to add a new argument to pass the path in the function `writeTDX`. I am not sure if it is a good method. Let me know your thoughts on it.

Reply:
I'm very confused why you write the file one place and move it elsewhere. You can make a `makeWriteTdx(root string) func(string, string, []byte, map[int]bool) error` and create your RtmrSubsystem with `WriteAttr: makeWriteTdx(tempDir)` to stay within the expected signature if you need. I just don't think it's safe to use a location that persists beyond the lifetime of the test, like os.TempDir().

Reply:
Thank you for the comments. Done.